examples/generator/railjson_generator/schema/infra/infra.py

```diff
@@ -20,6 +20,7 @@ class Infra:
     operational_points: List[OperationalPoint] = field(default_factory=list)
     tvd_sections: List[TVDSection] = field(default_factory=list)
     routes: List[Route] = field(default_factory=list)
+    speed_limits: List[Dict] = field(default_factory=list)
```

Comment:
Do we need to specify the label of a speed limit, we could generate it from speed limits in track sections.